src/server.py

```diff
@@ -1,28 +1,39 @@
 #!/usr/bin/env python
 
 from flask import Flask, request
-from newspaper import Article, fulltext
-import os, json
+from newspaper import Article, fulltext, Config
+import os, json, re
 
 app = Flask(__name__)
 import logging
 log = logging.getLogger('werkzeug')
 log.setLevel(logging.ERROR)
 
+linkedinPostUrl = 'https://www.linkedin.com/posts'
+
 @app.route('/', methods = ['GET'])
 @app.route('/topimage',methods = ['GET'])
 def api_top_image():
     url = request.args.get('url')
-    article = get_article(url)
+
+    config = Config()
+    if url.startswith(linkedinPostUrl):
```

Comment:
I would use a boolean variable for linked in posts

Reply:
how and why?

Reply:
```suggestion
    isLinkedinPost = url.startswith(linkedinPostUrl)
    if isLinkedinPost:
    
```